src/gui/transferlistmodel.h

```diff
@@ -39,6 +40,47 @@ namespace BitTorrent
     class TorrentHandle;
 }
 
+// convenience class to read text colors from qss since qss doesn't work with qobject
```

Comment:
Maybe try to inherit TransferListModel from QWidget too?

Reply:
I don't think multiple inheritance is allowed

Reply:
I actually did spend almost an hour to figure out what's happening since Qt docs never mention this

https://www.man42.net/blog/2011/09/qt-stylesheet-custom-q_property/